core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/aggregate/AggregationOperation.scala

```diff
@@ -52,6 +53,7 @@ class AggregationOperation {
 
   @JsonProperty(value = "result attribute", required = true)
   @JsonPropertyDescription("column name of average result")
+  @NotNull(message = "result attribute is required")
```

Comment:
Why don't we need size here?

Reply:
resultAttribute does not need @Size because @NotNull alone is sufficient to ensure it’s not null for a String.